RubeesEat/Startup.cs

```diff
@@ -27,15 +29,15 @@ public void ConfigureServices(IServiceCollection services)
     public void ConfigureServicesEAT(IServiceCollection services)

```

Comment:
In this method we should only configure the production setup. So no more in-memory stuff but only the database. For the tests, we override this default configuration in the test setup. The `ConfigureServices` method should also refrain from doing too much validation as services might be overriden from the tests. As such, please change the `SqlConnectionFactory` to take an `IConfiguration` object and a connection string name. The actual connection string should be lazy evaluated when the first connection is requested.